pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -852,6 +856,15 @@ func (gce *GCECloud) createTargetPool(name, serviceName, region string, hosts []
 	for _, host := range hosts {
 		instances = append(instances, makeHostURL(gce.projectID, host.Zone, host.Name))
 	}
+	if len(instances) > maxInstancesPerTargetPool {
```

Comment:
Hmm, we don't have to wait for node healthy api right? I mean just get this function (https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/service/servicecontroller.go#L629) to only choose the first 1000 healthy nodes, unless I'm reading that code wrong